ingest/ingest_pipeline.py

```diff
@@ -325,25 +323,32 @@ def ingest_expression(self) -> int:
             return 1
         return 0
 
+    # More work needs to be done to fully remove ingest from IngestPipeline
+    # Tracked in SCP-3023
     @custom_metric(config.get_metric_properties)
     def ingest_cell_metadata(self):
         """Ingests cell metadata files into Firestore."""
-        self.cell_metadata.preprocess()
+        validate_against_convention = False
+        if self.kwargs["validate_convention"] is not None:
```

Comment:
This nested `if` seems clunky. 
```suggestion
        if self.kwargs["validate_convention"] is not None and self.kwargs["validate_convention"]:
```

Reply:
0cde419

Reply:
That commit didn't appear to address this particular line.